components/typography/TextLink.js

```diff
@@ -0,0 +1,20 @@
+
+export default function TextLink({
```

Comment:
I prefer to name it `Href` or `Link` component :)

Reply:
There is already a `Link` component I think in master? And this is for applying the `rel="noopener noreferrer"` I believe?

Reply:
Yeah, also NextJS has `Link` component to make navigation between pages. Ok I see explanation about that name in other comment, we can resolve that discussion :)